packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -77,6 +79,8 @@ export class SwatchColorPickerBase extends BaseComponent<ISwatchColorPickerProps
       getStyles,
     } = this.props;
 
+    const htmlProps = getNativeProps(this.props, htmlElementProperties, ['aria-posinset, aria-setsize']);
```

Comment:
Do you need this here, could you just pass { ...this.props } to Grid and then do the trimming there?